src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -214,8 +222,16 @@ const ProjectSelector = createReactClass({
 
     return (
       <MenuItem {...menuItemProps}>
-        {project.isBookmarked && <span className="icon-star-solid bookmark " />}
-        {label}
+        {/*ideally an icon here...*/}
+        <ProjectBadge>
+          {!platforms || !platforms.length ? (
+            <EmptyPlatformicon />
+          ) : (
+            <PlatformList platforms={platforms} />
```

Comment:
We should move this into `app/components/idBadge/projectBadge` with another prop to control visibility because we will want this behavior elsewhere in the application.

It would be nice to have bookmark there too but that's outside of the scope of this PR.

I think we would probably want this order: `[star] [platformicons] [project name]`